src/components/base/ErrorPage.vue

```diff
@@ -0,0 +1,56 @@
+<template>
```

Comment:
I'm not sure I'm getting to this error page?

<img width="800" alt="Screen Shot 2022-01-05 at 1 29 37 PM" src="https://user-images.githubusercontent.com/5270855/148269755-568865ee-638d-4852-afb4-c1460d6aa10f.png">

Reply:
are you still getting this after the general router fix?

Reply:
yup - seems like the rewrite rule isn't working... 🤔

Reply:
after much head-banging, this seems to be working now